WordPress/Classes/Models/Revisions/RevisionDiff.swift

```diff
@@ -86,3 +86,21 @@ extension RevisionDiff {
         }
     }
 }
+
+
+extension RevisionDiff {
```

Comment:
Feels like to many extensions in this one file. Can we also move this one elsewhere?

Reply:
Yep! I was thinking the same yesterday about move the CoreData extensions to another file